src/server/services/data-transform.service.test.js

```diff
@@ -475,91 +475,409 @@ describe("data-transform.service transformAnswersForSummary()", () => {
       });
-    });
-    describe("Given that only no_import_export is part of cumulative answers", () => {
-      const cumulativeFullAnswers = {
-        no_import_export: "True"
-      };
-      it("Should return a import_export_activities value of 'None'", () => {
-        result = transformAnswersForSummary(cumulativeFullAnswers);
-        expect(result.import_export_activities).toBe("None");
+
+      describe("establishment null tests", () => {
+        describe("establishment address line 1 is equal to street but organisation is not present", () => {
+          const cumulativeAnswersEstAddSelected = {
+            establishment_address_selected: "0"
+          };
+
+          const testAddressLookupData = {
+            establishment_postcode_find: [
+              {
+                addressline1: "Example House",
+                addressline2: "Example Street",
+                addressline3: "Example Dependant Locality",
+                organisation: "",
+                street: "Example Street"
+              }
+            ]
+          };
+          const correctResponse = {
+            establishment_address_line_1: "Example House",
+            establishment_address_line_2: "Example Street",
+            establishment_address_line_3: "Example Dependant Locality"
+          };
+
+          it("returns the response with the organisation included as address line 1", () => {
+            const response = transformAnswersForSummary(
+              cumulativeAnswersEstAddSelected,
+              testAddressLookupData
+            );
+            expect(response.establishment_address_line_1).toBe(
+              correctResponse.establishment_address_line_1
+            );
+            expect(response.establishment_address_line_2).toBe(
+              correctResponse.establishment_address_line_2
+            );
+            expect(response.establishment_address_line_3).toBe(
+              correctResponse.establishment_address_line_3
+            );
+          });
+        });
+
+        describe("establishment address line 1 is not equal to street but organisation is present", () => {
+          const cumulativeAnswersEstAddSelected = {
+            establishment_address_selected: "0"
+          };
+
+          const testAddressLookupData = {
+            establishment_postcode_find: [
+              {
+                addressline1: "Example House",
+                addressline2: "Example Street",
+                addressline3: "Example Dependant Locality",
+                organisation: "The Organisation",
+                street: "Example Street"
+              }
+            ]
+          };
+          const correctResponse = {
+            establishment_address_line_1: "Example House",
+            establishment_address_line_2: "Example Street",
+            establishment_address_line_3: "Example Dependant Locality"
+          };
+
+          it("returns the response with the organisation included as address line 1", () => {
```

Comment:
the text description isn't accurate - "returns the response with the no changes to the address lines"